root/installBinary.sh

```diff
@@ -3,6 +3,8 @@
 . /plex-common.sh
 
 echo "${TAG}" > /version.txt
+echo "${PLEX_BUILD}" > /plex-build.txt
+echo "${PLEX_DISTRO}" > /plex-distro.txt
```

Comment:
I'm starting to wonder if this should instead be a single file that contains three variables.  When there was only one file, it was fine to have only that single variable but now that there are 3, maybe a single file containing the three variables would make more sense.  To make such a change, it looks like https://github.com/plexinc/pms-docker/blob/master/root/etc/cont-init.d/50-plex-update would also need some updating but that would be it.

Reply:
Yes, you are definitely right. I can try to merge these three variables into one file, then adapt the following scripts: `installBinary.sh`, `plex-common.sh`, `50-plex-update`